src/python/zquantum/vqe/utils.py

```diff
@@ -1,64 +1,61 @@
-from openfermion.utils import uccsd_singlet_paramsize
+from zquantum.core.circuit import Circuit, Gate, Qubit
+from forestopenfermion import exponentiate
+from openfermion import (
+    jordan_wigner,
+    bravyi_kitaev,
+    FermionOperator,
+    InteractionOperator,
+)
+from typing import Union
 
-def build_circuit_template(ansatz_type, n_mo, n_alpha, n_beta, transformation='Jordan-Wigner',
-                           fermion_generator=None, spin_ordering='interleaved', 
-                           n_qubits=None, ordering=None, layers=None,
-                           has_rx_layer=False, elementary=False):
-    """Constructs a circuit template for an ansatz.
+
+def exponentiate_fermion_operator(
+    fermion_generator: Union[FermionOperator, InteractionOperator],
+    transformation: str = "Jordan-Wigner",
+) -> Circuit:
+    """
+    Create a pyQuil circuit corresponding to the exponentiation of an operator.
```

Comment:
The summary should start on first line and this function returns a zquantum.core circuit. Not a pyQuil circuit

Reply:
Adressed in [75170d8](https://github.com/zapatacomputing/z-quantum-vqe/pull/5/commits/75170d8c5c38ec0adc9f66321ca231a9fec080cf).